src/main/java/org/mitre/synthea/modules/LifecycleModule.java

```diff
@@ -175,6 +193,50 @@ public static void birth(Person person, long time) {
     attributes.put(Person.SEXUAL_ORIENTATION, orientation);
   }
   
+  @SuppressWarnings("unchecked")
+  private static String fakeFirstName(String gender, String language, Random random) {
+    List<String> choices;
+    if ("spanish".equalsIgnoreCase(language)) {
+      choices = (List<String>) names.get("spanish." + gender);
+    } else {
+      choices = (List<String>) names.get("english." + gender);
+    }
+    // pick a random item from the list
+    return choices.get(random.nextInt(choices.size()));
+  }
+  
+  @SuppressWarnings("unchecked")
+  private static String fakeLastName(String language, Random random) {
+    List<String> choices;
+    if ("spanish".equalsIgnoreCase(language)) {
+      choices = (List<String>) names.get("spanish.family");
+    } else {
+      choices = (List<String>) names.get("english.family");
+    }
+    // pick a random item from the list
+    return choices.get(random.nextInt(choices.size()));
+  }
+  
+  @SuppressWarnings("unchecked")
+  private static String fakeAddress(boolean includeLine2, Random random) {
+    int number = random.nextInt(1000) + 100;
+    List<String> n = (List<String>)names.get("english.family");
```

Comment:
We'll have to fix this long-term. Doesn't make sense. There are so many Main Streets and "Oak St" and stuff like that... not too many "Johnson Lanes" out there, is my guess.

Reply:
Agreed, but note that this is actually roughly the approach faker uses as well:
https://github.com/DiUS/java-faker/blob/master/src/main/resources/en.yml#L34

They also include first names, but still they only use the set of people's names and nothing street-specific.